packages/vaadin-grid/test/helpers.js

```diff
@@ -268,3 +254,7 @@ export const fire = (type, detail, options) => {
   node.dispatchEvent(event);
   return event;
 };
+
+export const nextResize = (target) => {
+  return new Promise((resolve) => new ResizeObserver(() => setTimeout(resolve)).observe(target));
+};
```

Comment:
Created an issue to extract this to `@vaadin/testing-helpers`: https://github.com/vaadin/testing-helpers/issues/5